<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repayment Schedule Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cancelPaymentButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel Payment</string>
    </property>
   </widget>
   <widget class="QTableView" name="repaymentScheduleTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1171</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>68</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(247, 247, 247)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Repayment Schedule</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="toExcelButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>70</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>to Excel</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detailsButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paidButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
